README.md

```diff
@@ -29,6 +29,28 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Adding Throttling
+
+If the test environment is too good, and the metrics are too big to catch dips in performance after a PR you can test it with `CPU throttling` of 2 with the commands:

```

Comment:
You can simulate a low-end device with a throttling option. Available values are... (please explain the options and also the throttling means CPU throttling etc.) 

CC @daniel-stoian-lgp